recipes/tsne/setup.py

```diff
@@ -0,0 +1,91 @@
+"""
```

Comment:
Is this file actually needed?

Reply:
I've removed the file and will update when @danielfrg has released the next version.